backend/service/k8s/clientset.go

```diff
@@ -64,10 +70,29 @@ func newClientsetManager(rules *clientcmd.ClientConfigLoadingRules) (ClientsetMa
 		if err != nil {
 			return nil, err
 		}
-
 		lookup[name] = &ctxClientsetImpl{Interface: clientset, namespace: ns, cluster: ctxInfo.Cluster}
 	}
 
+	// If there is no configured cluster produced fallback to InClusterConfig
+	if len(lookup) == 0 {
+		logger.Warn("no kubeconfig was found, falling back to InClusterConfig")
```

Comment:
i think this should be info or debug as it may be a normal part of someone's startup.